source/driver-bulk-update.rst

```diff
@@ -2,43 +2,11 @@
 Bulk API Spec
 =============
 
-:Authors: Christian Kvalheim
 :Status: Deprecated
-:Type: Standards
-:Last Modified: May 18, 2021
+:Minimum Server Version: 2.4
```

Comment:
I inferred this from references to the server version elsewhere in the spec. IIRC, the bulk update spec was written before the actual write commands were implemented in 2.6 (which is what the related "server write commands" spec uses). That said, it probably doesn't matter too much since this spec is deprecated.

Reply:
2.4 sounds reasonable, but I agree it doesn't really matter both because this spec is deprecated and because no one is supporting 2.4 or 2.6 anymore so the distinction isn't meaningful. I'm hoping we will delete this spec altogether and move any info that is still relevant into the new bulk spec via DRIVERS-716.